recipes/openmc-plotter/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "openmc-plotter" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/openmc-plotter-{{ version }}.tar.gz
+  sha256: 06e9e17f12f119e81b9fbf80cee9276cf3ec144a6be9f783b304646d357c32b4
+build:
+  entry_points:
+    - openmc-plotter=openmc_plotter.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - matplotlib-base
+    - numpy
+    - openmc >0.12.2
+    - pyside2
+    - python  >=3.6
+
+test:
+  imports:
+    - openmc_plotter
+  commands:
+    - pip check
```

```suggestion
    - pip check
    - openmc-plotter --help
```
To check the entry point.

---

Ahh ok. Was it one of the following errors? I cannot see the logs anymore for some reason.

https://conda-forge.org/docs/maintainer/maintainer_faq.html#mfaq-qt-load-xcb

---

Interesting 🤔
Then please revert the CLI test and I’ll merge it.

---

I will revert it- the issue is actually just with argparse, not qt. It seems to go deeper into the code with `--help` than normal on CI because it is looking for input files to be run even with just `--help` (input files that are required from the user when the tool is used).